sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -1,49 +1,35 @@
 <template>
-  <v-navigation-drawer app class="body-background">
-    <v-list dense nav>
+  <v-navigation-drawer app class="body-background" permanent>
+    <v-list density="compact" nav>
```

Comment:
what does `density="compact"` do ?

Reply:
Basicly makes the menu and items "smaller". It looks more like Figma IMO.
The left one below is the "compact". Just a small difference.
![image](https://user-images.githubusercontent.com/61259237/176769557-eb68636d-2c60-4314-b002-f1c3f60bdb5f.png)